Libraries/Text/RCTTextView.m

```diff
@@ -70,6 +73,18 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
   return _backedTextInput;
 }
 
+- (void)fontAttributesDidChangeWithFont:(UIFont *)font
```

Comment:
(We can consider moving this to superclass.)

Reply:
This overrides the superclass's implementation of `fontAttributesDidChangeWithFont`. Are you suggesting I move this logic to the superclass's implementation so `RCTTextView` doesn't have to override it?

Reply:
Yes. `invalidateContentSize` does not exist in base class, right? I think we can also move this method to the base class (even with possibly empty implementation).

Reply:
(This does not block landing. We can do it in the future.)